doc/modules/ROOT/pages/ops.adoc

```diff
@@ -4,11 +4,11 @@ This file is _generated_ by #'nrepl.impl.docs/-main
 ////
 = Supported nREPL operations
 
-[small]#generated from a verbose 'describe' response (nREPL v0.6.1-SNAPSHOT)#
+[small]#generated from a verbose 'describe' response (nREPL v0.7.0-SNAPSHOT)#
 
 == Operations
 
-=== `:clone`
+=== `"clone"`
```

Comment:
I think here we shouldn't print them with quotes, as that looks a bit weird. `clone` would be fine.

Reply:
Done.